### PR TITLE
Mark CI/CI Task as Done

### PR DESCRIPTION
**Pull request by davidzech**
@Anh-KNguyen Hi can you review this?

### CODE REVIEW

File: main_test.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ import (
 )
 
 func TestNotes(t *testing.T) { // Test Suite is called TestNotes
+	t.FailNow()
```
Comment (Anh-KNguyen) [inline main_test.go]: remove this

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ We are going to create a basic web application with:
 ## Roadmap
 - [x] Backend
 - [x] Frontend
-- [ ] Continuous Integration and Testing with GitHub Actions
+- [x] Continuous Integration and Testing with GitHub Actions
```
Comment (Anh-KNguyen) [inline README.md:12]: are you sure this is done?

**Review by Anh-KNguyen — CHANGES_REQUESTED** [review]

File: main_test.go (hunk 1 above)
Comment (davidzech) [reply main_test.go]: Fixed!

File: README.md (hunk 2 above)
Comment (davidzech) [reply README.md:12]: Not yet!

**Review by Anh-KNguyen — APPROVED** [review]

**Pull request merged**